### ISSUE TITLE
Serratus-Lite summarizer script

### ISSUE THREAD

**Issue by TWilson1012** [ababaian/serratus#278]
I have a quick question about the summary report output from Serratus Lite for the nucleotide alignment. When I run the code provided, switching only the paths to the files and input fastq file, I get a summary file that contains most of the summary statistics, but does not include the section at the end with FASTA hits, and the file does not begin with the SUMZER COMMENT even when I specify that it should (see attached screenshot of a summary report). 

I have run the summarizer code on the ERR2756788.bam file to check if the issue was related to my input files, but in this case I am having the same issues with the summary file (i.e. it lacks the FASTA section and comment section, unlike the summary report in the example). 

Code Example: 

SUMZER_COMMENT=$(echo sra="na",genome="cov3ma",version=200818,date=$(date +%y%m%d-%R))
summarizer="python3 /home/tess/Serratus/serratus_summarizer.py /dev/stdin /home/tess/Serratus/cov3ma.sumzer.tsv ERR.summary /dev/null"
samtools view ERR2756788.bam | $summarizer

I am wondering if this is an issue with my code, or perhaps the FASTA portion of the report is only when SRA files are used for input? 

Thank you for the help! 
<img width="578" alt="Screenshot 2024-05-14 at 8 43 35 AM" src="https://github.com/ababaian/serratus/assets/169809097/7a7fa54b-9df9-49a4-b9c8-96c46602716d">

**Comment by ababaian** [comment]
Ah are you referring to [This documented section?](https://github.com/ababaian/serratus/wiki/.summary-Reports#fasta-section).

I think we removed that as it was supposed to be a "peak" into the data which was then made obsolete when we started storing `.bam` files. I would not recommend using this feature, instead retain the `.bam` files and open them in `IGV`, this is by all accounts a superior means of inspecting fasta reads and alignments. I'll update the documentation to remove that FASTA section. Sorry about the confusion.
